python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1317,12 +1320,40 @@ def test_dropna_inplace(ray_df, pd_df):
     ray_df.dropna(thresh=2, inplace=True)
     pd_df.dropna(thresh=2, inplace=True)
 
-    ray_df_equals_pandas(ray_df, pd_df)
+    assert ray_df_equals_pandas(ray_df, pd_df)
 
     ray_df.dropna(axis=1, how='any', inplace=True)
     pd_df.dropna(axis=1, how='any', inplace=True)
 
-    ray_df_equals_pandas(ray_df, pd_df)
+    assert ray_df_equals_pandas(ray_df, pd_df)
+
+
+@pytest.fixture
+def test_dropna_multiple_axes(ray_df, pd_df):
+    assert ray_df_equals_pandas(
+        ray_df.dropna(how='all', axis=[0, 1]),
+        pd_df.dropna(how='all', axis=[0, 1])
+    )
+    assert ray_df_equals_pandas(
+        ray_df.dropna(how='all', axis=(0, 1)),
+        pd_df.dropna(how='all', axis=(0, 1))
+    )
+
+
+@pytest.fixture
+def test_dropna_multiple_axes_inplace(ray_df, pd_df):
+    ray_df = ray_df.copy()
+    pd_df = pd_df.copy()
+
+    ray_df.dropna(how='all', axis=[0, 1], inplace=True)
+    pd_df.dropna(how='all', axis=[0, 1], inplace=True)
+
+    assert ray_df_equals_pandas(ray_df, pd_df)
+
+    ray_df.dropna(how='all', axis=(0, 1), inplace=True)
```

Comment:
create a new copy of the original `df`s first, else this operates on the result of the previous `dropna` call.

Reply:
Just noticed that I made this same bug above in `test_dropna_inplace`!